packages/aws-cdk/test/api/hotswap/hotswap-deployments.test.ts

```diff
@@ -8,11 +8,23 @@ let mockGetEndpointSuffix: () => string;
 
 beforeEach(() => {
   hotswapMockSdkProvider = setup.setupHotswapTests();
-  mockUpdateLambdaCode = jest.fn();
+  mockUpdateLambdaCode = jest.fn().mockReturnValue({});
   mockUpdateMachineDefinition = jest.fn();
   mockGetEndpointSuffix = jest.fn(() => 'amazonaws.com');
   hotswapMockSdkProvider.stubLambda({
     updateFunctionCode: mockUpdateLambdaCode,
+  }, {
+    // these are needed for the waiter API that the Lambda service hotswap uses
+    api: {
+      waiters: {},
+    },
+    makeRequest() {
+      return {
+        promise: () => Promise.resolve({}),
+        response: {},
+        addListeners: () => {},
+      };
+    },
```

Comment:
I think we can remove these changes from here, right? With the changes you made to `stubLambda()`?

Reply:
ah missed this one :)